requirements.txt

```diff
@@ -1,3 +1,4 @@
 dateparser==1.0.0
 psutil==5.8.0
 PyYAML==5.4.1
+requests==2.25.1
```

Comment:
requirements-notifications.txt might be better place. infact, it already has it.

Reply:
Will remove it from requirements.txt